index.js

```diff
@@ -22,24 +24,16 @@ function GitHub(options) {
     return new GitHub(options);
   }
 
-  this.options = options || {};
-  this.defaults = utils.defaults(this.options);
-
-  if (!this.options.apiurl) {
-    this.options.apiurl = 'https://api.github.com';
-  }
-
-  this.interpolate = function(opts) {
-    var data = extend({}, this.options, opts);
-    return utils.interpolate(opts.path, data);
-  }.bind(this);
+  this.options = typeof options === 'object' ? options : {};
```

Comment:
oh yea. maybe better would be just `options && typeof options === 'object'`
I think it's always better to do type checking and I do it everywhere.